extra/innochecksum.cc

```diff
@@ -100,6 +101,8 @@ FILE*				log_file = NULL;
 /* Enabled for log write option. */
 static bool			is_log_enabled = false;
 
+/* Skip freed pages */
+static bool			skip_freed_pages;
```

Comment:
The comment before the variable declaration is not adding any value, and it is not in Doxygen format either.

In my previous review, I quoted the redundant initialization of `is_log_enabled` with the thought that you’d remove all redundant zero-initialization of any global variables in this compilation unit.

I also quoted some code in `innochecksum_get_one_option()` that seemed to be redundant. You addressed that comment by not adding further redundant code there, but you did not remove other redundant code:
```c++
	case 'n':
		no_check = true;
		break;
	case 'a':
	case 'S':
		break;
	case 'w':
		do_write = true;
		break;
	case 'l':
		is_log_enabled = true;
		break;
```
Note: the following is not redundant in the same way, because the parameter that is bound to the `D` option is `&page_dump_filename`. For now, I am not suggesting to remove the variable `page_type_dump`, although I would assume that it is redundant too: `page_type_dump == !!page_dump_filename`.
```c++
	case 'D':
		page_type_dump = true;
		break;
```

Reply:
My intention is that I don't want to touch the old code. That's the reason, I didn't remove it. Since you insist, I will remove it in the upcoming patch.